etools-searchable-multiselection-menu.html

```diff
@@ -659,7 +691,8 @@
         },
 
         _singleValueSelected: function() {
-          this.options.forEach(function(item) {
+          var options = this.twoLines ? this.twoLinesOptions: this.options;
```

Comment:
Good catch, this is from a previous version. Also, the behavior you described above when searching and the checkmark disappearing is not exclusive to this PR, its happening in the other demos as well so its a bug with the entire component.